packages/bpk-component-map/package.json

```diff
@@ -13,11 +13,13 @@
     "react": "^15.0.0 || ^16.0.0"
   },
   "dependencies": {
+    "bpk-mixins": "^17.10.1",
+    "bpk-react-utils": "^2.6.1",
+    "prop-types": "^15.5.8",
     "react-google-maps": "^9.4.5"
   },
   "devDependencies": {
-    "bpk-component-icon": "^3.21.1",
-    "bpk-component-tooltip": "^3.1.15",
-    "prop-types": "^15.5.8"
+    "bpk-component-spinner": "^2.2.31",
+    "bpk-component-text": "^1.0.61"
```

Comment:
One question: should `react-google-maps` peer-dependencies (`@types/googlemaps`, `@types/markerclustererplus`, `@types/react`) be added as `devDependencies` to isolate the clients from those internal details?

Reply:
I'm not sure - the thing is those peer dependencies are all TypeScript definition packages, which aren't needed here. 

It would be good to not have the warning that they aren't there, but I'm not sure that we should be installing packages we aren't using just for the sake of that.

Reply:
If it means that we swallow the errors as opposed to multiple consumers having to deal with them then it might be the lesser of two evils.

We can even follow up with PR to address.

Reply:
> We can even follow up with PR to address.

@matthewdavidson you mean follow up with a PR back to `react-google-maps`?

Reply:
Yeah, if it bugs us so much :) We cant be the only ones that don't use typescript

Reply:
Yeah sounds good. I'm gonna leave it out of this PR though because scope creep, if you don't mind. I'll follow up with another after this.